src/compat/browser_compatibility_types.ts

```diff
@@ -96,10 +96,10 @@ const MediaSource_ : typeof MediaSource|undefined =
   win.MSMediaSource;
 
 const MediaKeys_ : ICompatMediaKeysConstructor|undefined =
-  win.MediaKeys ||
+  win.MSMediaKeys ||
   win.MozMediaKeys ||
   win.WebKitMediaKeys ||
-  win.MSMediaKeys ||
+  win.MediaKeys ||
```

Comment:
Huh? Why MSMediaKeys above all?

Reply:
Also we could break a lot of things without knowing by putting MediaKeys below all like that.